File: ex/ex1/Worker.java
public class Worker extends Officer {
    int degree;

    public Worker(String name, boolean sex, String address, int degree) {
        super(name, sex, address);
        this.degree = degree;
    }
    @Override
    public String toString() {
        return super.toString() + " " + this.degree;
    }
}

File: C#/Practice Project/PracticeProject1/Worker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PracticeProject1
{
	internal class Worker : Officer
	{
		public int Grade { get; set; }
		public Worker(string name, int age, bool sex, string address, int grade) : base(name, age, sex, address)
		{
			Grade = grade;
		}

		public override string ToString()
		{
			return base.ToString() + "\nGrade: " + Grade;
		}
	}
}
